css/light-styles.css

```diff
@@ -4,23 +4,23 @@
    ========================================================================== */
 @import url('http://fonts.googleapis.com/css?family=Lato');
 html, body {
-	font-family: 'Lato', sans-serif;
-  background-color: #22222;
+	font-family: "source-sans-pro","Helvetica Neue",Helvetica,Arial,sans-serif;
```

Comment:
This font choice. Leave that as it was before.
![screenshot from 2017-04-23 20-24-44](https://cloud.githubusercontent.com/assets/12448084/25314515/0f5e07fe-2835-11e7-8f6e-68c964317813.png)
![screenshot from 2017-04-23 20-24-54](https://cloud.githubusercontent.com/assets/12448084/25314517/16d0bac2-2835-11e7-9cf8-83ce09314009.png)